modules/nw-about-multicast.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * networking/using-multicast.adoc
+
+[id="about-multicast-{context}"]
+= About multicast
+
+With IP multicast, data is broadcast to many IP addresses simultaneously.
+
+[IMPORTANT]
+====
+At this time, multicast is best used for low bandwidth coordination or service
+discovery and not a high-bandwidth solution.
+====
+
+Multicast traffic between {product-title} pods is disabled by default. If you
+are using the *OpenShiftSDN* network plug-in, you can enable multicast on a
+per-project basis.
+
+When using the *OpenShiftSDN* network plug-in:
+
+. Multicast packets sent by a pod will be delivered to all other pods in the
```

Comment:
I think these should be regular bullets, because it's not a set of steps or something that happens in a particular order.